client/components/post-schedule/clock.jsx

```diff
@@ -38,17 +38,24 @@ function checkTimeValue( value ) {
 }
 
 class PostScheduleClock extends Component {
+	constructor( props ) {
+		super( props );
+
+		// bounds
+		this.setTime = this.setTime.bind( this );
```

Comment:
For this one method you could use Stage 2 class property initializers:

``` js
class PostScheduleClock extends Component {
    setTime = () => {
        // ...
    };

    // ...
}
```
More food for thought than anything else